[draft-27] Django Admin Setup

aarajh opened the pull request:
**What?**

- Enabled django admin and setup database for development.
- Added support for our custom models in django admin.
- Updated main and /server readme files including better guide

**Why?**
- To be able to view and manage users from Django Admin GUI.

adil1508 commented on README.md:
```diff
@@ -0,0 +1,40 @@
+
```
ooh i like this new Readme

aaraj is v nyc <3

adil1508 commented:
Can we rebase off of master? 

I don't know why Github can't sigh. It doesn't look very conflicting to me :(

aarajh commented:
> 
> 
> The only comment i have is whether we need the binary files under
> 
> Server/app/app/**pycache**/
> Server/app/core/**pycache**/
> Server/app/core/management/**pycache**/
> Server/app/core/management/commands/**pycache**/
> 
> or should these directories be in the .gitignore?

True we dont. Fixed now. Added python .gitignore and removed the unnecessary files

adil1508 commented on Server/app/core/admin.py:
```diff
@@ -1,3 +1,33 @@
+from core import models
+from django.utils.translation import gettext as _
 
-# Register your models here.
```
Why u hate this line :'(

aarajh replied:
Because it was in the auto-template :p

adil1508 commented:
lgtm 🚀
